src/util/sss_semanage.c

```diff
@@ -459,7 +456,7 @@ int sss_del_seuser(const char *login_name)
     sss_semanage_close(handle);
     return ret;
 }
-#else /* HAVE_SEMANAGE && HAVE_SELINUX */
```

Comment:
I think the comment should not be removed, but modified to reflect the new situation (/* HAVE_SELINUX */). This `else` is about 400 lines away from its corresponding `if`, making it hard to know at a single glance what it is doing.

Reply:
ok, added.